tests/security/selinux/selinux_setup.pm

```diff
@@ -32,7 +32,16 @@ sub run {
     }
 
     # program 'sestatus' can be found in policycoreutils pkgs
-    zypper_call("in policycoreutils policycoreutils-python");
+    zypper_call("in policycoreutils");
+    my $ret = script_run('zypper -n in policycoreutils-python');
+    if ($ret) {
+        if ($ret == 104) {
+            record_soft_failure 'bsc#1116288 - package policycoreutils-python not found in module SLE-Module-Development-Tools';
+        }
+        else {
+            die "NEW Package installation failed";
```

Comment:
And it would `die` except 104?
Also the string describtion is not clear, it should tell what pacakage install failed

Reply:
I want it to die except 104 and 0.
Added package name in die message.